polaris-core/src/main/java/org/apache/polaris/core/persistence/impl/PolarisMetaStoreManagerImpl.java

```diff
@@ -1006,6 +1012,20 @@ public Map<String, String> deserializeProperties(PolarisCallContext callCtx, Str
         : new PrincipalSecretsResult(secrets);
   }
 
+  @Override
+  @Nonnull
+  public ResolutionManifestBuilder newResolutionManifestBuilder(
```

Comment:
This should be completely separate from the `PolarisMetaStoreManager`. I think the `Resolver` process is business logic, not persistence and I think the `PolarisMetaStoreManager` interface should be focused on persistence only.